framework/jaif-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
I thought they would use it on Cygwin. I changed it to /bin/sh.
What other things could I do to make this script more compatible with Windows?